change/@azure-node-token-validation-7f710590-0a83-448e-9a03-d398955429f8.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Should this be none?
```suggestion
  "type": "none",
```